cogs/displayinfo.py

```diff
@@ -0,0 +1,116 @@
+import discord
+import constantvariables
+import mongoDB
+from discord.ext import commands
+from discord.ext import tasks
+from datetime import datetime
+from datetime import date
+import datetime as dt
+from pymongo.mongo_client import MongoClient
+from pymongo.server_api import ServerApi
+import asyncio
+
+
+
+
+# Create a new client and connect to the MongoDB server
+mongoClient = MongoClient(mongoDB.MONGODB_URI, server_api=ServerApi('1'))
+
+
+try:
+    # Send a ping to confirm a successful connection to MongoDB
+    mongoClient.admin.command('ping')
+    print("Pinged your deployment. You successfully connected to MongoDB!")
+except Exception as e:
+    print(e)
+
+# Select the database
+db = mongoClient.Rasenbot
```

Comment:
This shouldn't be hard-coded right? Since it's only selecting the database name?

Reply:
The code is currently creating more collections for each server all under the same database "Rasenbot". So I believe it is fine hardcoding that as I'm storing all the data under "Rasenbot" database.